test_examples.py

```
@@ -0,0 +1,130 @@
+    print(stdout)
+    if stderr.strip():
+        print("--- STDERR ---")
+        print(stderr)
+
+    # Determine if the test was successful:
+    # 1. If it timed out, but printed output indicating success (e.g. Connected: True, Candle, Balance, Yield)
+    # 2. If it exited cleanly with return code 0
+    # 3. If it failed with a known credentials/session issue (since we might have a expired/different session or not set up email/password)
+    # 4. If it has a syntax or import error, it's a failure.
+    if timed_out:
+        print(f"Process timed out (10s) as expected for streaming examples.")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)